ynput/ayon-usd-resolver
# Usd Resolver Fixes for slow OsPath Reselution on windows and a Layer muting issue.

## Changelog Description
this PR fixes slow OsPath Resolves on Windows. 
it also Fixes an SDF-ARGS issue that caused an Layer muting issue where lower layers where not correctly identified and thereby not Muted. 

## Testing notes:
1. Try opening the ALab project in Houdini with and without the AyonUsdResolver (this should be only a tiny bit slower than the native UsdResolver) 

1. generate a usd scene using the contribution workflow. 
2. load the scene and see if you can mute nested layers that have SDF-ARGS
